packages/cli/src/cli/build.rs

```diff
@@ -55,6 +57,8 @@ impl Build {
             }
         }
 
+        plugins_after_command(BuildEvent).await;
```

Comment:
It might make sense to move this after the `index.html` file is generated so that plugins can modify that file. I'm thinking of asset plugins like the tailwind plugin automatically reaching in and checking that you have a tailwind file imported somewhere in your `<head>`